airbyte-webapp/src/components/JobItem/JobItem.tsx

```diff
@@ -63,24 +63,25 @@ export const JobItem: React.FC<JobItemProps> = ({ shortInfo, job }) => {
   };
 
   useEffectOnce(() => {
-    if (linkedJobId) {
-      scrollAnchor.current?.scrollIntoView({
-        behavior: "smooth",
-        block: "start",
-      });
+    if (linkedJobId === String(getJobId(job))) {
+      // We need to wait here a bit, so the page has a chance to finish rendering, before starting to scroll
+      // since otherwise this scroll won't really do anything.
+      const timeout = window.setTimeout(() => {
+        scrollAnchor.current?.scrollIntoView({
+          behavior: "smooth",
+          block: "start",
+        });
+      }, 1000);
+      return () => {
+        window.clearTimeout(timeout);
+      };
     }
+    return undefined;
```

Comment:
https://reactjs.org/docs/hooks-reference.html#uselayouteffect
?

Reply:
This unfortunately doesn't help (I barely ever had a case where useLayoutEffect actually helped :D). This will just cause it to slip into the execution slot after the next rendering, which unfortunately isn't enough in this case. There is just too much (async) rendering still ongoing, that even setting a timeout of like 300ms isn't enough to get this work. Tested also with `useLayoutEffect` and it does neither scroll (without the high timeout).

Reply:
@krishnaglick https://eslint.org/docs/latest/rules/consistent-return but sometimes it's tricky with use effect calls. Alternative I do:

```ts
if (linkedJobId !== String(getJobId(job))) {
  return;
}

// ...The actual code that returns a function here...

return () => ...
```

Reply:
I don't think it's a _better_ idea, but could do a `setInterval(() => {}, 100)` that runs until it's loaded.

Reply:
We unfortunately also don't have a way to determine whether it's loaded in that sense. The DOM reference IS there from thebeginning and mounted, so there is no clear way to determine that the browser is done with rendering everything around it.

Reply:
We do!
![Screen Shot 2022-08-08 at 4 17 11 PM](https://user-images.githubusercontent.com/4967694/183506706-1c266b87-b523-4940-b9d0-eb636e785f11.png)
![Screen Shot 2022-08-08 at 4 17 29 PM](https://user-images.githubusercontent.com/4967694/183506752-6f76a5a0-cd29-4a08-bcd1-16f96167d680.png)

Can fiddle with exact implementation, but I think this is a nice solution no?

Reply:
Oh good call. Yeah we def can use the animationComplete there to trigger the scroll to. Will change this tomorrow.

Reply:
I can push this if you'd like

Reply:
Pushed to `kc/fix-link-to-logs`

Reply:
Added this to this PR, as discussed offline this won't work in Chrome (on Linux) apparently, but since it'll work in Firefox stable, we'll still keep this code in.